shared/modules/selectors/util.js

```diff
@@ -1,9 +1,9 @@
 import { TransactionStatus } from '@metamask/transaction-controller';
 import { isEqual } from 'lodash';
-import { createSelectorCreator, defaultMemoize } from 'reselect';
+import { createSelectorCreator, lruMemoize } from 'reselect';
 
 export const createDeepEqualSelector = createSelectorCreator(
-  defaultMemoize,
+  lruMemoize,
```

Comment:
Just to clarify, this implementation is the same under the hood right?

Reply:
Yes! It was just renamed in v5.

Reply:
- Changelog: https://github.com/reduxjs/reselect/releases/tag/v5.0.1
- PR: https://github.com/reduxjs/reselect/pull/654